README.md

```diff
@@ -27,11 +27,9 @@ CodeMirror 6 has [great documentation](https://codemirror.net/6/docs/). The [sys
 
 The project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-If you use Visual Studio Code you might find the devcontainer included in this repository a useful option to get an automatically configured development environment. [Find out more about containers and Visual Studio Code](https://code.visualstudio.com/docs/remote/containers).
```

Comment:
I'll happily reinstate and update this dev container if anyone gets in touch to say they use it. Removing it for now as, so far as we know, it's unused by folks working on this project and therefore untested/unmaintained.